torchci/pages/api/drci/drci.ts

```diff
@@ -11,17 +11,24 @@ interface PRandJobs {
     jobs: RecentWorkflowsData[];
 }
 
+export default async function handler() {
+    fetchWorkflows();
+}
+
 export async function fetchWorkflows() {
     const recentWorkflows: RecentWorkflowsData[] = await fetchRecentWorkflows(
         NUM_MINUTES + ""
     );
 
     const workflowsByPR = reorganizeWorkflows(recentWorkflows);
 
-    for (const [pr_number, pr] of workflowsByPR) {
-        const { pending, failedJobs } = getWorkflowAnalysis(pr);
-        const failureInfo = constructFailureAnalysis(pending, failedJobs, pr.head_sha);
-        //add failureInfo to Dr. CI comment on each PR
+    for (const [pr_number, pr_info] of workflowsByPR) {
+        const { pending, failedJobs } = getWorkflowAnalysis(pr_info);
+
+        const failureInfo = constructFailureAnalysis(pending, failedJobs, pr_info.head_sha);
+        const comment = formDrciComment(pr_number) + failureInfo + DRCI_COMMENT_END;
```

Comment:
oooh nvm i see, the formDrciComment is more like formDrciHeader --> maybe renaming the function would be clearer.